requirements.txt

```diff
@@ -35,3 +35,6 @@ slacker==0.7.0
 # Documentation requirements
 Sphinx==1.3.1
 sphinx-autobuild==0.5.2
+
+# Journalism Workflow requirements
```

Comment:
should this live in a journalism-requirements.txt, or a requirements.txt in the example_project?